tox.ini

```diff
@@ -29,5 +29,12 @@ commands = pytest tests/ui/test_discopane.py {posargs}
 deps = flake8==3.3.0
 commands = flake8 {posargs:.}
 
+[testenv:dennis-lint]
+deps = dennis==0.9
+# Run via `bash -c` to make sure the file globbing are properly resolved on the bash
```

Comment:
s/file globbing are properly resolved on the bash/file globbing is properly resolved in the shell/